examples/avm/exportTx-ant-cchain.ts

```diff
@@ -57,8 +57,8 @@ const main = async (): Promise<any> => {
     let assetID: Buffer = utxo.getAssetID()
     const outputidx: Buffer = utxo.getOutputIdx()
     let secpTransferOutput: SECPTransferOutput = new SECPTransferOutput()
-    if(avaxAssetID.toString("hex") === assetID.toString("hex")) {
-      assetID = avaxAssetID
+    if(avaxAssetIDBuf.toString("hex") === assetID.toString("hex")) {
+      assetID = avaxAssetIDBuf
```

Comment:
I might be missing something since I don't know typescript very well, but is this a no-op? We're checking if `avaxAssetIDBuf` is the same as `assetID`, and if they are, we set `assetID` to `avaxAssetIDBuf`, but aren't they already the same?

Reply:
Good catch. Resolved.